api/tests/prescription/campaign/integration/infrastructure/repositories/campaign-report-repository_test.js

```diff
@@ -539,8 +547,8 @@ describe('Integration | Repository | Campaign-Report', function () {
 
         // then
         expect(campaignsWithReports[0]).to.be.instanceof(CampaignReport);
-        expect(campaignsWithReports[0]).to.deep.include(
-          _.pick(campaign, [
+        expect(campaignsWithReports[0]).to.deep.include({
+          ..._.pick(campaign, [
```

Comment:
Le spread n'est pas nécessaire
```suggestion
        expect(campaignsWithReports[0]).to.deep.include(
          _.pick(campaign, [
```